### ISSUE TITLE
Add landuse=residential to our world_basemap

### ISSUE THREAD

**Issue by sonora** [osmandapp/Osmand#1445]
Most people are amazed by the quality of our world_basemap, having a great world overview at their fingertips with just a single download map. The one thing which cannot be judged well on this map is the extent of buildup areas (cities, towns), because we do not include landuse=residential. While we e.g. depict forested areas very well by showing the green area, the gray for residential is missing, I think adding it down to zoom=10 would be a notable improvement.

**Comment by vshcherb** [comment]
I already saw it around the small villages.

**Comment by xmd5a2** [comment]
It's already in basemap but filtered by area.

**Comment by sonora** [comment]
Yes, we include from minzoom 6. I have added it to default.render to test it. It does not overall help much. Maybe working just with landuse=residential is not the right approach anyway, only a so-so workaround. What I am trying to achieve is that city areas are somehow marked better by a conventional gray area (we mark all forests, but no city areas, which are equally or more important for travelers), instead cities are just marked by the labeled dot, and you see the accumulation of roads a bit. Almost any city in the world an serve as a example, see one screenshot attached.

![city-area-invisible](https://cloud.githubusercontent.com/assets/845735/9080073/a0adf33c-3b50-11e5-971a-54bbaa5b3791.png)

**Comment by vshcherb** [comment]
I don't see grey area around that city http://www.openstreetmap.org/relation/3374767#map=13/48.7150/44.5274

**Comment by sonora** [comment]
Certainly true. there is no visible area, only relation 3374767. And it is questionable if we should use something like that to color city backgrounds in our world map. But if we used this to create a gray background, it would make a cleaner effect.

I only picked a random example of how city areas appear in our world map (essentially an uncolored area with many disconnected roads), while other (printed) maps on that scale would somehow show a gray area indicating that this is a built-up/city area. If we somehow managed to paint this area (like we do well for all forested areas), I think our world map would find eve more user acceptance. But I have no idea or concept what to use or how to proceed here for now.
